ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template2.kt

```diff
@@ -85,30 +77,31 @@ private fun Template2MainContent(state: PaywallViewState.Template2, viewModel: P
         horizontalAlignment = Alignment.CenterHorizontally,
         verticalArrangement = Arrangement.spacedBy(UIConstant.defaultButtonVerticalSpacing, Alignment.CenterVertically),
     ) {
-        IconImage(paywallData = state.paywallData)
-        val localizedConfig = state.paywallData.localizedConfig()
+        state.templateConfiguration.images.iconUri?.let { IconImage(it) }
+        val localizedConfig = state.templateConfiguration.packageConfiguration.defaultInfo.localization
```

Comment:
One thing I'ver done in iOS is have some helpers to make it easier to access some of this data that's deep down.
For example this could be `state.templateConfiguration.selectedLocalization`.

Reply:
Added something like that in https://github.com/RevenueCat/purchases-android/pull/1242/commits/e49b5a50cf13ace04ee9138dbfb7cdb114fed2b1